Packs/TAXIIServer/Integrations/TAXII2Server/TAXII2Server.py

```diff
@@ -63,7 +63,7 @@ def __init__(self, url_scheme: str, host: str, port: int, collections: dict, cer
             types_for_indicator_sdo: The list of stix types to provide indicator stix domain objects.
         """
         self._url_scheme = url_scheme
-        self._host = host
+        self._host = host.replace('.xdr.', '.crtx.')
```

Comment:
Is that relevant in all cases?

Reply:
Per the [docs](https://github.com/demisto/content/blob/master/Packs/TAXIIServer/Integrations/TAXII2Server/TAXII2Server_description.md#how-to-access-the-taxii-service) as I understood yes.
LMK if you think different.
<img width="1706" alt="Screenshot 2024-09-08 at 23 53 53" src="https://github.com/user-attachments/assets/425b4cb1-d331-4913-a8a5-a80d6e2bfba4">